README.md

```diff
@@ -32,13 +34,113 @@ Go 1.5 users will need to set `GO15VENDOREXPERIMENT=1` to get vendored dependenc
 Upon successful build the plugin binary will be available in `build/sriov`. 
 
 ## Enable SR-IOV
-
+### Intel cards
 Given Intel ixgbe NIC on CentOS, Fedora or RHEL:
 
 ```
 # vi /etc/modprobe.conf
 options ixgbe max_vfs=8,8
 ```
+### Mellanox cards
+SRIOV-CNI support Mellanox ConnectX�-4 Lx and ConnectX�-5 adapter cards.
+To enable SR-IOV functionality the following steps are required:
+
+1- Enable Virtualization (SR-IOV) in the BIOS.
+
+add ```intel_iommu=on``` to ```/boot/grub/grub.conf```:
+
+```
+# cat /boot/grub/grub.conf
+...
+kernel /vmlinuz-Ub16.04x86_64-4.4.0-135-generic root=/dev/sda2 console=tty0 console=ttyS0,115200n8 rhgb intel_iommu=on
+...
+```
+
+2- Enable SR-IOV in the NIC's Firmware.
```

Comment:
Is the process here the same on different distros? for example, fedora, rhel etc

Reply:
yes it is the same for all distros except for Enable Virtualization (SR-IOV) in the BIOS

Reply:
Thanks!